docs/developer-guide-commands.md

```diff
@@ -0,0 +1,128 @@
+# Development Helpers
+
+This document serves as a reference point to helper commands available under
+this project.
+
+## Building
+
+```shell
+# If pkg/apis/ has been changed, run generator to update client code
+make generate
+
+# Build binaries of all components, make format is called as a part of it
+make build
+
+# Refresh vendoring after dependencies change
+make dep
+```
+
+## Testing
+
+```shell
+# Check code formatting, imports and properly generated modules
+make check
+```
+
+## Containers
+
+```shell
+# Build containers of all components, binaries are built as a part of multi-stage containerized build
+make docker
+
+# Push built containers to remote registry
+make docker-push
+```
+
+It is possible to adjust container images with following environment variables.
+
+```shell
+IMAGE_REGISTRY # quay.io/nmstate
+IMAGE_TAG # latest
+STATE_HANDLER_IMAGE # kubernetes-nmstate-state-handler
+POLICY_HANDLER_IMAGE # kubernetes-nmstate-configuration-policy-handler
+```
+
+## Manifests
+
+Manifests in `manifests/examples/` folder are built from templates kept in
+`manifests/templates/`.
+
+```shell
+# build manifests
+make manifests
+```
+
+Manifest templates contain following variables. It it possible to adjust them
+my setting environment variables before calling `make manifests`.
+
+```shell
+NAMESPACE # nmstate-default
+IMAGE_REGISTRY # quay.io/nmstate
+IMAGE_TAG # latest
+PULL_POLICY # Always
+STATE_HANDLER_IMAGE # kubernetes-nmstate-state-handler
+POLICY_HANDLER_IMAGE # kubernetes-nmstate-configuration-policy-handler
+```
+
+You can also specify input and output directories.
+
+```shell
+MANIFESTS_SOURCE # manifests/templates
+MANIFESTS_DESTINATION # manifests/examples
+```
+
+## Local Cluster
+
+This project uses [kubevirtci](https://github.com/kubevirt/kubevirtci) to
+deploy local cluster.
+
+Use following commands to control it.
```

Comment:
Use *the* following commands to control it:

(End with a colon and add the article)

Reply:
Done